tools/postgres/diff_schema.js

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-extraneous-dependencies */
+const program = require("commander");
+const randomstring = require("randomstring");
+const execSync = require("child_process").execSync;
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+const tmp = require("tmp");
+const rimraf = require("rimraf");
+const replace = require("replace-in-file");
+
+let POSTGRES_URL = process.env.POSTGRES_URL;
+const ORIGINAL_POSTGRES_URL =
```

Comment:
From a code style POV, I think we should get rid of `ORIGINAL_POSTGRES_URL` and use regular variables or function parameters in case we need to derive it.